conf/migrations.py

```diff
@@ -35,3 +35,15 @@ def migration_231129_deploy_workflow(settings, data):
             logger.info(
                 f'Migrated {product_type}.DEPLOY_WORKFLOW to {product_type}.DEPLOY_WORKFLOWS'
             )
+
+
+def migration_241120_http_proxy_ipv6_url(settings, data):
+    """Migrates server.http_proxy_ipv6_url to http_proxy.http_proxy_ipv6_url"""
+    if (
+        settings.server.get('http_proxy_ipv6_url')
+        and isinstance(settings.server.http_proxy_ipv6_url, str)
+        and not settings.http_proxy.get('http_proxy_ipv6_url')
+    ):
+        data['http_proxy'] = {}
+        data['http_proxy'].http_proxy_ipv6_url = settings.server.http_proxy_ipv6_url
```

Comment:
Optional:

I know you copied that from the migration I wrote a couple of months back, but I see a small improvement in terms of readability. Note that `data` is a `Box` object already.

```suggestion
        data.http_proxy = {}
        data.http_proxy.http_proxy_ipv6_url = settings.server.http_proxy_ipv6_url
```

one might wish getting rid of assigning the empty dict as well, but that is needed:

```python
[ins] In [35]: data = Box()

[ins] In [36]: data.http_proxy.http_proxy_ipv6_url = "foo"
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
File ~/.virtualenvs/robottelo/lib64/python3.12/site-packages/box/box.py:593, in box.box.Box.__getitem__()

KeyError: 'http_proxy'

The above exception was the direct cause of the following exception:

BoxKeyError                               Traceback (most recent call last)
File ~/.virtualenvs/robottelo/lib64/python3.12/site-packages/box/box.py:632, in box.box.Box.__getattr__()

File ~/.virtualenvs/robottelo/lib64/python3.12/site-packages/box/box.py:620, in box.box.Box.__getitem__()

BoxKeyError: "'http_proxy'"

During handling of the above exception, another exception occurred:

AttributeError                            Traceback (most recent call last)
File ~/.virtualenvs/robottelo/lib64/python3.12/site-packages/box/box.py:634, in box.box.Box.__getattr__()

AttributeError: 'Box' object has no attribute 'http_proxy'

The above exception was the direct cause of the following exception:

BoxKeyError                               Traceback (most recent call last)
Cell In[36], line 1
----> 1 data.http_proxy.http_proxy_ipv6_url = "foo"

File ~/.virtualenvs/robottelo/lib64/python3.12/site-packages/box/box.py:648, in box.box.Box.__getattr__()

BoxKeyError: "'Box' object has no attribute 'http_proxy'"

[ins] In [37]: data.http_proxy = {}

[ins] In [38]: data.http_proxy.http_proxy_ipv6_url = "foo"
```

Reply:
updated the MR